examples/life.htm

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<style>
+		.flatbutton
+		{
+			width:		200px;
+			margin:		0 auto;
+			text-align:	center;
+			font-family:	Arial;
+			font-size:	16px;
+			font-weight:	bold;
+			cursor:		pointer;
+		}
+	</style>
+	<head>
+		<title>Life via qKit</title>
+	</head>
+	<body>
+		<script src="qkit.js"></script><!-- https://raw.githubusercontent.com/fahadhaidari/qKit/master/qkit.js -->
+		<script src="life.js"></script>
+		<br/>
+		<div align="center" class="flatbutton" style="background-color:#FFC0C0;" 
+			onclick="Random5050Habitat();RenderCurrentPhase();">Reset to 50/50 Random</div>
+		<br/>
+		<div align="center" class="flatbutton" style="background-color:#FFFFC0;" 
+			onclick="PopulateOneWalker();RenderCurrentPhase();">Populate One Walker</div>
+		<br/>
+		<div align="center" class="flatbutton" style="background-color:#C0FFC0;" 
+			onclick="ComputeNextPhase();phase++;RenderCurrentPhase();">Step</div>
+		<br/>
+		<div align="center" class="flatbutton" style="background-color:#C0C0FF;" 
+			onclick="if(auto){auto=false;this.innerHTML='Run';}else{auto=true;this.innerHTML='Pause';}">Pause</div>
+		<br/>
+		<div align="center" class="flatbutton" style="background-color:#D0D0D0;" 
+			onclick="if(scroll){scroll=false;this.innerHTML='Scroll';phase%=2;RenderCurrentPhase();}else{scroll=true;this.innerHTML='Center';phase%=2;}">Center</div>
+	</body>

```

Comment:
In the examples directory, there are no `.html` files, only `.js` so please try to have all of your code in `life.js`

Reply:
This is a duplicate response: [Will comply.  As a side-note, the idea behind the HTM was to provide an interface to showcase features built in to the JS.  Do you have any interest in a convention or annex for that?  Surely there'd be a traffic-jam if each JS came with its variety of index.html.]